charts/cluster-api-cluster-openstack/README.md

```diff
@@ -49,28 +50,25 @@ spec:
           region: en-west-1
```

Comment:
Note to self: `ca` I fixed, so it resolves automatically and shouldn't be documented, `region` doesn't exist.